<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1091</width>
    <height>696</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>866</width>
    <height>696</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1091</width>
    <height>696</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IT Equipment Reservations</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons.qrc">
    <normaloff>:/Icons/clock104.png</normaloff>:/Icons/clock104.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="deliveriesTable">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>721</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="newReservationBtn">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>40</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>New Reservation</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Deliveries : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>340</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Pickups :</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="pickupsTable">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>360</y>
      <width>721</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pickupRetrievedBtn">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>600</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>View / Retrieve</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editReservation">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>90</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit Reservation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="viewDetailsButton">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>140</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>View All Details</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deliveredButton">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>280</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Delivered</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="deliveryNotePreview">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>211</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="pickupNotePreview">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>211</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Note Preview : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Note Preview : </string>
    </property>
   </widget>
   <widget class="QLabel" name="connectionIndicator">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>10</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1091</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionAdmin_Panel"/>
    <addaction name="actionForce_Update"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdmin_Panel">
   <property name="text">
    <string>Admin Panel</string>
   </property>
   <property name="whatsThis">
    <string extracomment="Super secret admin access"/>
   </property>
  </action>
  <action name="actionForce_Update">
   <property name="text">
    <string>Force Update</string>
   </property>
   <property name="whatsThis">
    <string extracomment="Forces the application to initiate an update"/>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
